src/agent/onefuzz-supervisor/src/work.rs

```diff
@@ -67,6 +67,13 @@ impl WorkSet {
 
         Ok(())
     }
+
+    pub fn setup_dir(&self) -> Result<PathBuf> {
```

Comment:
Thinking about the future, we'll want some struct that represents the OneFuzz filesystem context, and this method should live on that, then compute the setup dir for some work parmeter (relative to itself). But this is good for now.